css-three-column-layout/styles.css

```diff
@@ -0,0 +1,81 @@
+.row {
+  display: flex;
+  flex-basis: auto;
+  justify-content: center;
+  background-color: #1996da;
+  padding-bottom: 25px;
+}
+
+h2 {
+  display: block;
+  text-align: center;
+  justify-content: center;
+  opacity: 0.9;
+}
+
+.white-text {
+  color: white;
+}
+
+img {
+  background-color: white;
+  width: 50%;
+}
+
+.font {
+  font-family: Lato, sans-serif;
+}
+
+p {
+  margin: auto;
+  width: 75%;
+  opacity: 0.6;
```

Comment:
This is making the element itself more see-through, this is not ideal. Instead, you likely just need to adjust the text color.